### PR TITLE
Adds support for mapping DCSS Weather Request/Response interactions

### PR DESCRIPTION
**Pull request by michaelrfraser**
- DCSS WeatherRequestLocation is mapped to DcssWeatherRequestPdu
- DCSS GroundResponse is mapped to DcssWeatherResponsePdu
- DCSS CloudLayerResponse is mapped to DcssWeatherResponsePdu
- Adds support for datatypes introduced in the above interactions
- Added helper methods to AbstractMapper for serializing/deserializing
  interaction parameters

### CODE REVIEW

**Comment by michaelrfraser** [comment]
NOTE: Corresponding unit tests to come

**Comment by michaelrfraser** [comment]
Unit test have been completed and committed

**Comment by michaelrfraser** [comment]
Merging this as I need it downstream for other work. Tim can shoot me later if he has a problem with it :)

File: codebase/src/java/disco/org/openlvc/disco/connection/rpr/custom/dcss/interactions/GroundResponse.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	//----------------------------------------------------------
+
+	//----------------------------------------------------------
+	//                   INSTANCE VARIABLES
+	//----------------------------------------------------------
+	private HLAfloat32BE precipitationRate;
+	private HLAfloat32BE temperature;
+	private HLAfloat32BE humidity;
+	private HLAfloat32BE pressure;
+
```
Comment (timpokorny) [inline codebase/src/java/disco/org/openlvc/disco/connection/rpr/custom/dcss/interactions/GroundResponse.java:37]: Just testing something. Ignore for now.

**Pull request merged**